stable/artifactory/values.yaml

```diff
@@ -39,6 +39,8 @@ serviceAccount:
 
 ingress:
   enabled: false
+  defaultBackend:
```

Comment:
Please add this to `artifactory-ha` as well. We like to keep them aligned where possible.

Reply:
done in 9e2a15175f76af2e5baba908b28f9d07329e4ffa